doc/version_history.rst

```diff
@@ -6,6 +6,27 @@
 Version History
 ###############
 
+v0.2.1
+------
+
+Changes:
+
+* Work around a chiller bug that causes the chiller to hang with "MODE MISMATCH ALARM" displayed on the panel, requiring power cycling.
+* `ATWhiteLightCsc`: fix a bug whereby the CSC could try to fault when already in fault state.
+* `ChillerModel`: fix incorrect values reported for alarms and warnings.
+  Do this by reversing alarm and warning data strings before parsing them as hexadecimal values.
+  See the note in `lsst.ts.idl.enums.ATWhiteLight.ChillerL1Alarms` for more information.
+* `ChillerClient`: fix a log message that claimed to show a reply but actually showed the command.
+* Remove the unused ``examples`` directory.
+
+Requires:
+
+* ts_idl
+* ts_salobj 7.1
+* ts_tcpip
+* ts_utils
+* IDL file for ATWhiteLight built from ts_xml 11.1
```

Comment:
I noticed you had a PR on ts_xml for ATWhiteLight. Shouldn't this therefore be version `12.1` instead?

Reply:
Those are good reasons to keep it at 11.1.